pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/Microsoft.NETCore.UniversalWindowsPlatform.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <Version>5.2.3</Version>
+    <Version>5.2.4-beta</Version>
```

Comment:
For the versioning for UWP metapackage, I would defer to @weshaggard and @ericstj.

In general, you are adding something fundamentally new, so updating the patch version is not sufficient. At minimum, we need to update the minor version.

Also, when do you expect to have this version of the package published and who is going to consume it and how?